lib/android/build.gradle

```diff
@@ -67,11 +67,4 @@ dependencies {
   implementation "com.github.skyscanner:backpack-android:${_backpackAndroidVersion}"
   implementation "androidx.constraintlayout:constraintlayout:1.1.3"
   implementation "com.jakewharton.threetenabp:threetenabp:${rootProject.ext.threetenabpVersion}"
-
-  if (_internalBuild) {
```

Comment:
As we are using the `react-native` dark mode implementation we no longer need to bundle the third party dep as its no longer used in the app 🎉